topics/sequence-analysis/tutorials/ncbi-fcs/tutorial.md

```diff
@@ -0,0 +1,193 @@
+
+
+> <agenda-title></agenda-title>
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Retrieving the data
+
+FCS operates on assembled genome sequences and is not intended for use on raw reads. The following tutorial uses an assembled genome from yeast (*Saccharomyces cerevisiae*) with contaminants artificially inserted into the genome. The first step is to retrieve the genome FASTA. 
+
+## Upload the genome FASTA from NCBI FCS GitHub
+
+The following steps provide instructions to upload the test dataset into your Galaxy instance.
+> <hands-on-title> Data Upload </hands-on-title>
+>
+> **Step 1:** Create a new history for this tutorial
+>
+>    > {% snippet  faqs/galaxy/histories_create_new.md %}
+>
+> **Step 2:** Copy the datasets URLs into clipboard.
+>     Click on the copy button in the right upper corner of the box below.
+>    ```
+>    https://zenodo.org/records/10881551/files/FCS.combo.test.fa
+>    ```
+>
+> **Step 3:** Upload `fasta.gz` dataset into Galaxy
+>    > {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>
+>    > {% snippet faqs/galaxy/datasets_import_from_data_library.md %}
+>
+{: .hands_on}
+
+## Confirm dataset upload
+
+Your data should look like this:
+
+![Galaxy History with Uploaded Data](../../images/ncbi-fcs/fcs-galaxy-1.jpg "Figure 1. Galaxy History with Uploaded Data.")
+
+# Importing Workflows
+
+Next we will import a Galaxy workflow - a chain of tools to perform a set of operations on a user-supplied input. Specificially in this workflow, we will:
+
+1. Screen the genome for foreign organism sequences using **FCS-GX** `screen` mode.
+1. Produce a cleaned set of contigs using **FCS-GX** `clean` mode.
+1. Screen the genome for synthetic sequences using **FCS-adaptor** and remove identified contaminants. 
+1. Produce a final set of cleaned contigs using a second iteration of **FCS-GX** `clean` mode.
+
+![NCBI Foreign Contamination Screen Galaxy Workflow](../../images/ncbi-fcs/fcs-galaxy-2.jpg "Figure 2. NCBI Foreign Contamination Screen Galaxy Workflow.")
+
+> <hands-on-title> Importing Galaxy Workflows </hands-on-title>
+>
+> **Step 1:** Copy the workflow URLs into clipboard.
+>
+>    ```
+>    https://training.galaxyproject.org/training-material/topics/sequence-analysis/tutorials/ncbi-fcs/workflows/ncbi-fcs.ga
+>    ```
+>
+> **Step 2:** Open Galaxy.
+> 
+>    > <warning-title>Log in to Galaxy</warning-title>
+>    > Make sure you are logged in to your Galaxy account!
+>    {: .warning}
+>
+>
+> **Step 3:** Import NCBI Foreign Contamination Screen Workflow into your Galaxy interface.
+>    > {% snippet  faqs/galaxy/workflows_import.md %}
+>
+{: .hands_on}
+
+Once the workflow is loaded, your workflow page should look like this:
+
+![Galaxy Workflows page with NCBI Foreign Contamination Screen](../../images/ncbi-fcs/fcs-galaxy-3.jpg "Figure 3. Galaxy Workflows page with NCBI Foreign Contamination Screen.")
```

Comment:
If you accept the above change, you'll need to adjust this screenshot a bit, and remove the steps of launching the workflow since the user will already be in the simplified workflow run form interface.